decidim-comments/lib/decidim/api/commentable_mutation_type.rb

```diff
@@ -14,7 +14,7 @@ class CommentableMutationType < Decidim::Api::Types::BaseObject
       end
 
       def add_comment(body:, alignment: nil, user_group_id: nil)
-        params = { "comment" => { "body" => body, "alignment" => alignment, "user_group_id" => user_group_id, "commentable" => object } }
+        params = { "comment" => { "body" => body, "alignment" => alignment, "user_group_id" => user_group_id, "commentable" => object, "locale" => "en" } }
```

Comment:
Here may be an issue with that locale set hard coded to english ... maybe would better to be set to `context[:current_organization].default_locale` ?